docs/integrations/third-party-libraries.md

```diff
@@ -46,6 +46,7 @@ This can be done using third-party libraries created by the community.
 - [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
 - [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
+- [zustand-sync-tabs](https://www.npmjs.com/package/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)
```

Comment:
Do you have a Github link? It was hard to found quickly from the NPM page, at least for me.

Most if not all links here are repository links, so I think at this point this is what most people expect.